Mathlib/Algebra/Lie/OfAssociative.lean

```diff
@@ -254,6 +254,23 @@ theorem LieSubalgebra.toEndomorphism_mk (K : LieSubalgebra R L) {x : L} (hx : x
 
 variable {R L M}
 
+namespace LieModule
+
+variable {M₂ : Type w₁} [AddCommGroup M₂] [Module R M₂] [LieRingModule L M₂] [LieModule R L M₂]
+  (f : M →ₗ⁅R,L⁆ M₂) (k : ℕ) (x : L)
+
+lemma toEndomorphism_pow_compl_lieHom :
```

Comment:
What is `compl` here?

Reply:
(If it's `LinearMap.comp`, I think plain `comp` would be better)

Reply:
Renamed to use `comp` instead of `compl`.